jax/_src/nn/functions.py

```diff
@@ -518,6 +519,32 @@ def glu(x: ArrayLike, axis: int = -1) -> Array:
 logsumexp = _logsumexp
 
 
+@partial(api.jit, static_argnames=("axis", "keepdims"))
+def logmeanexp(
+    x: ArrayLike,
+    axis: int | tuple[int, ...] | None = None,
+    where: ArrayLike | None = None,
+    keepdims: bool = False,
+) -> Array:
+  r"""Log mean exp.
+  Computes the function:
+  .. math::
+    \text{logmeanexp}(x) = \log \frac{1}{n} \sum_{i=1}^n \exp x_i = \text{logsumexp}(x) - \log n
```

Comment:
This is leading to doc build errors:
```
/__w/jax/jax/jax/_src/nn/functions.py:docstring of jax._src.nn.functions.logmeanexp:4: ERROR: Unexpected indentation.
/__w/jax/jax/jax/_src/nn/functions.py:docstring of jax._src.nn.functions.logmeanexp:5: WARNING: Literal block ends without a blank line; unexpected unindent.
```
I think you need a blank line before and after the math block.